README.md

```diff
@@ -67,9 +93,10 @@ git config --global --unset url."http://".insteadOf
 ```
 
 _Notes:_ 
+ - If you are behind NTLM proxy you can use [Cntlm Authentication Proxy](http://cntlm.sourceforge.net/).
  - If you are on Windows replace `export` with `set` to set env variables
- - Replace `localhost:3128` with standard pattern `http://user:pwd@host:port`
- - Git configuration is needed because Bower use it for fetching from GitHub
+ - Replace `localhost:3128` with the standard pattern `http://user:pwd@host:port`
+ - For zeppelin-web: currently there is no way to reach Bower main repo through NTLM proxy
```

Comment:
Thanks @fvaleri for the documentation!
One question, is it really true that bower can not go through NTLM proxy?
I've found a related issue https://github.com/bower/bower/issues/1368